client/js/lounge.js

```diff
@@ -375,6 +375,7 @@ $(function() {
 				utils.toggleNotificationMarkers(false);
 			}
 
+			self[0].scrollIntoView({behavior: "smooth", block: "nearest", inline: "nearest"});
```

Comment:
Can you make this a helper function so that same parameters can also be used in userlist scrolling without having to duplicate them?

Reply:
Done. Still fighting wonk in my development environment (the scroll works if I set a breakpoint and step through the scroll function, but not if I just let the page run uninterrupted) but this should work.

Should I also convert the userlist code?